src/components/Overlay.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Tags from './Tags'
+import { ReactComponent as Arrow } from '../icons/back-arrow.svg'
+import { ReactComponent as Share } from '../icons/share.svg'
+import Contributors from './Contributors'
+import Detail from './Detail'
+
+export default function Overlay({ project, onBack }) {
+  return (
+    <OverlaySection>
+      <Headline>
+        <ArrowStyled onClick={onBack} />
+        <Title>{project.title}</Title>
+        <ShareStyled />
+      </Headline>
+      <Img alt='' src={project.image ?? 'https://source.unsplash.com/random'}></Img>
+      <TagList>
+        {project.tags.map((tag, index) => (
+          <Tags content={tag} key={index} />
+        ))}
+      </TagList>
+
+      <Contributors contributors={project.contributors} />
+      <Detail title='Mission' content={project.mission} />
+      <Detail title='About' content={project.about} />
+      <Detail title='Updates' content={project.updates} />
+      <Detail title='Comments' content={project.comments} />
```

Comment:
If the content property in project is only important for Detail component, you could put these in an array in your project object and map over it to display it.

Reply:
Done